src/Microsoft.DotNet.Cli.Utils/CommandResolution/DefaultCommandResolver.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.ProjectModel.Graph;
+using Microsoft.Extensions.PlatformAbstractions;
+using NuGet.Frameworks;
+using NuGet.Packaging;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class DefaultCommandResolver : CompositeCommandResolver
+    {
+        public static DefaultCommandResolver Create()
+        {
+            var environment = new EnvironmentProvider();
+            var packagedCommandSpecFactory = new PackagedCommandSpecFactory();
+
+            var platformCommandSpecFactory = default(IPlatformCommandSpecFactory);
```

Comment:
So... you are using a default to avoid putting the type in the declaration? Well, you already are, just not on the left side. I find using the default a little more convoluted and less natural then just declaring the type, in this case.